src/components/com_tjcertificate/administrator/models/certificates.php

```diff
@@ -194,6 +243,8 @@ protected function getListQuery()
 		$orderCol  = $this->state->get('list.ordering', 'ci.id');
 		$orderDirn = $this->state->get('list.direction', 'desc');
 
+		$query->group('ci.id');
```

Comment:
why is this needed?...this will affect the other certificate type which is added programmatically

Reply:
removed now used for tightly coupled